Module2_Webpack_Advanced/src/webpack/prod.js

```diff
@@ -0,0 +1,9 @@
+const webpackMerge = require('webpack-merge');
+
+const webpackBaseConfig  = require('./webpack.config.js');
+
+module.exports = () => {
+    let baseConfig = webpackBaseConfig;
+    console.log(baseConfig);
+    return baseConfig;
```

Comment:
as for me it is better to use webpackMerge plugin to merge common webpack config with particular(dev, prod, test..) webpack config

Reply:
used